CRVConditions/inc/CRVDigitizationPeriod.hh

```diff
@@ -0,0 +1,10 @@
+#ifndef CRVConditions_CRVDigitizationPeriod_hh
+#define CRVConditions_CRVDigitizationPeriod_hh
+
+namespace mu2e {
+
+  constexpr double CRVDigitizationPeriod = 12.55; // ns
```

Comment:
I would like to check this number. We have been using 12.55ns for a long time, but I want to check that it is not actually 12.50ns.

Reply:
It turns out that 12.55 is an older value, and should be updated to 12.5. 
@rlcee Could you please change it.